packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss

```diff
@@ -139,12 +140,12 @@
 
 .high,
 .low {
-  box-shadow: 0 0 0 1px #000000;
+  background-color: colors.$orange-20;
```

Comment:
I think this should be `colors.$orange-10` per the hex code in Ciaran's screenshot https://openmrs.atlassian.net/browse/O3-3656.

Reply:
Also, per the acceptance criteria in the ticket, you'd need to implement the hover state backgrounds.

Reply:
I've sought clarification from Ciaran in the issue thread.